packages/storage/__tests__/providers/AWSS3Provider-unit-test.ts

```diff
@@ -280,25 +273,19 @@ describe('StorageProvider test', () => {
 				on: jest.fn(),
 				removeAllListeners: jest.fn(),
 			};
-			jest
-				.spyOn(events, 'EventEmitter')
-				.mockImplementationOnce(() => mockEventEmitter);
+			jest.spyOn(events, 'EventEmitter').mockImplementationOnce(() => mockEventEmitter);
 			const downloadOptionsWithProgressCallback = Object.assign({}, options);
 			const storage = new StorageProvider();
 			storage.configure(downloadOptionsWithProgressCallback);
-			jest
-				.spyOn(S3Client.prototype, 'send')
-				.mockImplementationOnce(async params => {
-					return { Body: [1, 2] };
-				});
+			jest.spyOn(S3Client.prototype, 'send').mockImplementationOnce(async params => {
+				return { Body: [1, 2] };
+			});
 			await storage.get('key', {
 				download: true,
+				// @ts-ignore - Disable TS check here as this is intentional
 				progressCallback: 'this is not a function',
```

Comment:
In this case it is not a big deal, but `@ts-ignore` can mask other problems, that's why I would prefer something like:

```suggestion
        progressCallback: <S3ProviderGetConfig['progressCallback']><unknown>'this is not a function', // This is intentional
```